result/brace-style.js

```diff
@@ -0,0 +1,209 @@
+if (foo) {
+  bar();
+} else {
+  baz();
+}
+if (foo) {
+  bar();
+}
+else {
+  baz();
+}
+if (foo)
+{
+  bar();
+}
+else
+{
+  baz();
+}
+/*eslint brace-style: "error"*/
+
+function foo()
+{
+  return true;
+}
+
+if (foo)
+{
+  bar();
+}
+
+try
+{
+  somethingRisky();
+} catch(e)
+{
+  handleError();
+}
+
+if (foo) {
+  bar();
+}
+else {
+  baz();
+}
+/*eslint brace-style: "error"*/
+
+function foo() {
+  return true;
+}
+
+if (foo) {
+  bar();
+}
+
+if (foo) {
+  bar();
+} else {
+  baz();
+}
+
+try {
+  somethingRisky();
+} catch(e) {
+  handleError();
+}
+
+// when there are no braces, there are no problems
+if (foo) bar();
+else if (baz) boom();
```

Comment:
__[development][eslint]__
* Error - 'baz' is not defined. (no-undef)
* Error - 'boom' is not defined. (no-undef)